### ISSUE TITLE
Credentials Manager only works every other time

### ISSUE THREAD

**Issue by KosayJabre** [ForkIssues/TrackerWin#923]
This is driving me nuts and I can't find anything online about it. 

I am having an authentication issue with Fork and Git. I have tried rebooting, re-installing Fork and Git, and removing my credentials from Credential Manager.

When doing anything to remote on Fork, I get an authentication error. If I go to Credential Manager > edit my Git credentials > type in my password again, it works. I can then do one pull/push.

However, if I try to pull/push again, I get the same authentication error until I edit my credentials again.

So basically, when I edit my credentials, I get _one pull/push_, and then get authentication failed until re-editing again.

Im using git instance 2.26.windows.1.

**Comment by DanPristupov** [comment]
What git instance do you use (File -> Preferences -> Git)?

**Comment by KosayJabre** [comment]
@DanPristupov I am using 2.26.windows.1

**Comment by DanPristupov** [comment]
Could you check what `credential.helper` value you have in `%localappdata%\fork\gitInstance\2.26.2\etc\gitconfig`?
 
The correct value should be
 
```
[credential]
helper = manager
```

**Comment by KosayJabre** [comment]
Yep, I have that value. This is the full content of my file:

```
[core]
        symlinks = false
        autocrlf = true
        fscache = true
[color]
        diff = auto
        status = auto
        branch = auto
        interactive = true
[help]
        format = html
[diff "astextplain"]
        textconv = astextplain
[rebase]
        autosquash = true
[filter "lfs"]
        clean = git-lfs clean -- %f
        smudge = git-lfs smudge -- %f
        process = git-lfs filter-process
        required = true
[credential]
        helper = manager
```

**Comment by DanPristupov** [comment]
Did you try to change Fork git instance to the system one in preferences?

**Comment by DanPristupov** [comment]
What git service do you use?
Can you show what your remote address looks like? (don't forget to edit private info)

**Comment by KosayJabre** [comment]
I also have git instance 2.28 in preferences, and the error persists after switching to that.

**Comment by KosayJabre** [comment]
Currently using in-house hosted Gitlab service. Looks something like https://{hostname}.net/{project}/{repo}.git

**Comment by KosayJabre** [comment]
I'm using Fork 1.53.2

**Comment by DanPristupov** [comment]
This part is managed by git credential manager (GCM) which is part of the official git distribution (https://github.com/Microsoft/Git-Credential-Manager-for-Windows).

What you can do:

1. Open command line (Ctrl+Alt+T) and ensure that problem persists when you run `git fetch`
2. Set the env variables `GCM_TRACE=1` and `GIT_TRACE=1` and run `git fetch`. Redact any private information and attach the log

**Comment by PaulHeisterkamp** [comment]
I had the exact same problem and changing the gitconfig fixed it:
Previous value
[credential]
	helper = !\"C:/Users/Adminf4dd100e49/AppData/Local/Fork/gitInstance/2.24.1/mingw64/libexec/git-core/git-credential-manager.exe\"

Fixed value
[credential]
	helper = manager

**Comment by c-key** [comment]
Hello, I had the same problem and I found some information about it here: https://github.com/microsoft/Git-Credential-Manager-Core/issues/154
I think it is a bug in the credential manager core.

I changed the helper value under the credential section from **manager-core** to **manager** and all things work fine.
